src/Components/NavBar/NavBar.js

```diff
@@ -1,58 +1,133 @@
+                <ul className="navLinks">
+                    <li className="navLinkOuter">
+                        <Link className="navLink" to="/">
+                            Home
+                        </Link>
+                    </li>
+                    <li className="navLinkOuter">
+                        <Link className="navLink" to="/About">
+                            About
+                        </Link>
+                    </li>
+                    <li className="navLinkOuter">
+                        <Link className="navLink" to="/Episodes">
+                            Episodes
+                        </Link>
+                    </li>
+                    <li className="navLinkOuter">
+                        <Link className="navLink" to="/Projects">
+                            Projects
+                        </Link>
+                    </li>
+
+                    <li className="navLinkOuter">
+                        <Link className="navLink" to="/Contact">
+                            Contact
+                        </Link>
+                    </li>
+                </ul>
+                <a
+                    href="https://www.patreon.com/hoppedupeast"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                >
+                    <button className="patreonDonate">
+                        <img
+                            className="navPatreonLogo"
+                            src={patreon}
+                            alt="Patreon P icon"
+                        />
+                        <span>DONATE</span>
+                    </button>
+                </a>
+            </div>
+
+            <div className="navHamburger">
+                <Link to="/">
+                    <img
+                        src={hugeLogo}
+                        className="navbarHamburgerLogo"
+                        alt="Hopped Up Gaming:East Logo"
+                    />
+                </Link>
+                <button
+                    onClick={hamburgerClick}
+                    className="navbarHamburgerButton"
+                >
+                    <img
+                        src={hamburger}
+                        className="navbarHamburgerIcon"
+                        id="hamburgericon"
+                        alt="hamburger icon"
+                    />
+                </button>
+                <div id="hamburgerLinks" className="navHamburgerLinks">
+                    <Link
+                        className="hamburgerLink"
+                        onClick={hamburgerClick}
+                        to="/"
+                    >
+                        <span className="hamburgerLinkText">Home</span>
                     </Link>
-                </li>
-                <li className="navLinkOuter">
-                    <Link className="navLink" to="/About">
-                        About
+
+                    <Link
```

Comment:
you could do something similar as my suggestion above with the hamburger menu as well sincve each item in the menu is eseentially the same as eachother